templates/eap73-amq-persistent-s2i.json

```diff
@@ -0,0 +1,1057 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss",
+            "version": "7.3",
+            "openshift.io/display-name": "JBoss EAP 7.3 + AMQ 7 with passthrough TLS",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform application using Red Hat JBoss AMQ with persistence and secure communication using https. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/7.3.x/README.adoc",
+            "template.openshift.io/long-description": "This template defines resources needed to develop a JBoss Enterprise Application Platform based application, including a build configuration, application deployment configuration, using Red Hat JBoss AMQ with persistence and secure communication using passthrough TLS.",
+            "template.openshift.io/documentation-url": "https://access.redhat.com/documentation/en/red-hat-jboss-enterprise-application-platform/",
+            "template.openshift.io/support-url": "https://access.redhat.com"
+        },
+        "name": "eap73-amq-persistent-s2i"
+    },
+    "labels": {
+        "template": "eap73-amq-persistent-s2i",
+        "xpaas": "7.3"
+    },
+    "message": "A new JBoss EAP and AMQ persistent based application with SSL support has been created in your project. The username/password for accessing the AMQ service is ${MQ_USERNAME}/${MQ_PASSWORD}. Please be sure to create the following secrets: \"${HTTPS_SECRET}\" containing the ${HTTPS_KEYSTORE} file used for serving secure content; \"${JGROUPS_ENCRYPT_SECRET}\" containing the ${JGROUPS_ENCRYPT_KEYSTORE} file used for securing JGroups communications.",
+    "parameters": [
+        {
+            "displayName": "Application Name",
+            "description": "The name for the application.",
+            "name": "APPLICATION_NAME",
+            "value": "eap-app",
+            "required": true
+        },
+        {
+            "displayName": "EAP Image Name",
+            "description": "Name of the EAP image to use, example: jboss-eap73-openjdk11-openshift:7.3",
+            "name": "EAP_IMAGE_NAME",
+            "value": "jboss-eap73-openjdk11-openshift:7.3",
```

Comment:
Default value still points to regular EAP 7.3 (intel) image.
Is there a point to copy the templates for z-series? It seems to me it would be simpler to maintain and document if we keep using the existing EAP 7.3 templates and only mention in the doc that the `EAP_IMAGE_NAME` and `EAP_RUNTIME_IMAGE_NAME` MUST be set to the OpenJ9 image streams.

@luck3y wdyt?

Reply:
The reason for this change is that docs were updated for z/; to point to the new location (and there was a bit of a lack of clarity about who was creating the imagestream and where it was supposed to go.)

So I think we should move these templates now, and we can switch other EAP images over to using them as time allows.

Reply:
But to clarify, once these are moved, and the remaining docs updated, then the JDK 8 and 11 images can use these templates as well, the default will just be JDK 11 on AMD.